test/built-ins/String/prototype/padEnd/exception-not-object-coercible.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2016 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-string.prototype.padend
+description: >
+  String#padEnd should fail if given a null or undefined value,
+  or an object not coercible to a string.
+author: Jordan Harband
+---*/
+
+assert.throws(TypeError, function () {
+    String.prototype.padEnd.call(null);
+});
+
+assert.throws(TypeError, function () {
+    String.prototype.padEnd.call(undefined);
+});
+
+var notCoercible = {
+    toString: function () {
+        throw new Test262Error('attempted toString');
+    },
+    valueOf: function () {
+        throw new Test262Error('attempted valueOf');
+    }
```

Comment:
valueOf is not reachable. You already verify this on the observable order tests.